app/build.gradle

```diff
@@ -101,15 +97,15 @@ dependencies {
     implementation 'com.google.android.material:material:1.1.0'
 
     // Miscellaneous
-    implementation 'com.google.guava:guava:28.2-android'
+    implementation 'com.google.guava:guava:29.0-android'
```

Comment:
I will have a loot to the changelog first as it's a new major version.

Reply:
I looked through the changelings when making these changes. You are only using the `net` part, and nothing in net was changed.